client/app/email-domain/email/account/migrate/email-domain-email-account-migrate.controller.js

```diff
@@ -47,9 +47,12 @@ angular.module('App').controller(
       this.$scope.alerts.migrate = 'domain_alert_migrate';
       this.$scope.migrateAccount = () => this.migrateAccount();
 
-      // URLs constants
-      this.allGuides = _.get(this.urls, 'FR.guides.all');
-      this.emailsOrder = _.get(this.urls, 'FR.emailsOrder');
+      this.User.getUrlOf('guides').then((guides) => {
```

Comment:
You could chain the promises instead. Can't you get more than one URL at the same time ?